release-notes/docs/3.12.0.md

```diff
@@ -0,0 +1,117 @@
+# 3.12.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Breaking changes](#breaking-changes)
+- [Supported software](#supported-software)
+- [UI/UX changes](#uiux-changes)
+- [Highlights](#highlights)
+- [And more...](#and-more)
+
+## Known issues
+
+<<< TODO >>>
+
+## Upgrade notes
+
+- This upgrade can be rolled out remotely. Users will download the new version in the background and be prompted to reload the app when it's ready. A small amount of data will be needed to download the new version.
+- Upgrading to this version does not require other applications to be upgraded. This release does not drop support for any hardware or software that works with the previous version.
+- This release modifies some CouchDB view definitions, so we recommend you Stage the upgrade and wait for the views to be rebuilt before rolling it out to minimize downtime.
+- It is recommended to test the upgrade on a clone of your server first to ensure it works well for your application.
+
+### Breaking changes
+
+None.
+
+### Supported software
+
+There are no required changes to the [supported software matrix](https://docs.communityhealthtoolkit.org/core/overview/supported-software/) from `3.0.0`.
+
+| Node | CouchDB | Browsers | SMS bridge | Android | medic-android | medic-couch2pg |
+|----|----|----|----|----|----|---|
+| 8.11+ | 2.1+ | Chrome 53+, Firefox latest | medic-gateway | 4.4+ | 0.4.5+ | 3.0+ |
+
+### UI/UX changes
+
+None.
+
+## Highlights
+
+### Telemetry
+
+The CHT-Core is collecting new telemetry data:
+
+- [#6300](https://github.com/medic/cht-core/issues/6300): The difference between the client datetime and the server datetime, it helps to detect user's clock errors.
+- [#6915](https://github.com/medic/cht-core/issues/6915): Access to granular telemetry data for post-hoc analysis, monitoring & research initiatives.
+- [#6354](https://github.com/medic/cht-core/issues/6354): Access to replication information.
+
+### Monitoring
+
+The Monitoring API version 1 (`/api/v1/monitoring`) is now deprecated and the version 2 (`/api/v2/monitoring`) is introduced with new metrics available:
```

Comment:
Be sure to link to the documentation. Also it'd be worth explaining that the v1 API will continue to work so people aren't worried about a breaking change, but explain that we improved the return structure in the v2 version.

Reply:
Done!